schema/deploy/shipping/views.sql

```diff
@@ -2911,4 +2912,163 @@ comment on view shipping.uw_reopening_ehs_reporting_v1 is
   on shipping.uw_reopening_ehs_reporting_v1
   to "ehs-results-exporter";
 
+
+create or replace view shipping.__uw_priority_queue_v1 as (
+    with all_uw_instances as (
+        select
+              encounter_id,
+              individual.identifier as individual,
+              encountered::date as encountered,
+              encounter.details -> '_provenance' -> 'redcap' ->> 'project_id' as redcap_project_id,
+              encounter.details -> '_provenance' -> 'redcap' ->> 'record_id' as redcap_record_id,
+              encounter.details -> '_provenance' -> 'redcap' ->> 'event_name' as redcap_event_name,
+              encounter.details -> '_provenance' -> 'redcap' ->> 'repeat_instance' as redcap_repeat_instance
+        from warehouse.encounter
+        join warehouse.individual using (individual_id)
+        where encounter.details @> '{"_provenance": {"redcap": {"url": "https://redcap.iths.org/", "project_id":23854}}}'
+    ),
+
+    uw_enrollments as (
+        select
+            all_uw_instances.*,
+            tier
+        from all_uw_instances
+        join shipping.uw_reopening_enrollment_fhir_encounter_details_v1 using (encounter_id)
+        where redcap_event_name = 'enrollment_arm_1'
+    ),
+
+    uw_encounters as (
+        select
+            all_uw_instances.*,
+            (select boolean_response from shipping.fhir_questionnaire_responses_v1 where encounter_id = all_uw_instances.encounter_id and link_id = 'screen_positive') as screen_positive,
+            (select boolean_response from shipping.fhir_questionnaire_responses_v1 where encounter_id = all_uw_instances.encounter_id and link_id = 'daily_symptoms') as daily_symptoms,
+            (select case string_response[1] when 'yes' then true else false end from shipping.fhir_questionnaire_responses_v1 where encounter_id = all_uw_instances.encounter_id and link_id = 'daily_exposure') as daily_exposure,
+            (select case string_response[1] when 'yes' then true else false end from shipping.fhir_questionnaire_responses_v1 where encounter_id = all_uw_instances.encounter_id and link_id = 'daily_exposure_known_pos') as daily_exposure_known_pos,
+            (select trim(string_response[1]) from shipping.fhir_questionnaire_responses_v1 where encounter_id = all_uw_instances.encounter_id and link_id = 'testing_trigger') as testing_trigger,
+            (select trim(string_response[1]) from shipping.fhir_questionnaire_responses_v1 where encounter_id = all_uw_instances.encounter_id and link_id = 'surge_selected') as surge_selected,
+            sample.collected::date as sample_collection_date
+        from all_uw_instances
+        left join warehouse.sample using (encounter_id)
+        where redcap_event_name = 'encounter_arm_1'
+    ),
+
+    uw_individuals as (
+        select
+            individual,
+            max(uw_encounters.encountered) filter (where testing_trigger = '1') as latest_invite_date,
```

Comment:
I saw this and realized that testing_trigger should be a boolean. See https://github.com/seattleflu/id3c-customizations/pull/153